src/tests/ftest/util/daos_perf_base.py

```diff
@@ -36,7 +37,8 @@ def run_daos_perf(self):
         daos_perf.cont_uuid.update(self.container.uuid)
         daos_perf.dmg_config_file.update(dmg_config_file)
         self.log.info("daos_perf command: %s", str(daos_perf))
-        daos_perf_env = daos_perf.get_environment(self.server_managers[0])
+        daos_perf_env = daos_perf.env.copy()
+        daos_perf_env["D_LOG_FILE"] = get_log_file("{}_daos.log".format(daos_perf.command))
```

Comment:
I realize this is a cherry-pick, so I don't think we should change this now, but given the scope of the `daos_perf` object shouldn't his just be:
```suggestion
        daos_perf.env["D_LOG_FILE"] = get_log_file("{}_daos.log".format(daos_perf.command))
```
and then later:
```
        self.job_manager.assign_environment(daos_perf.env)
```

Reply:
You're right. I didn't notice that :)